### ISSUE TITLE
[enhancement] Logograph features

### ISSUE THREAD

**Issue by Fcomega121** [DraqueT/PolyGlot#649]
hello DraqueT! I've been wondering on your program and I think i've got some ideas to improve logographs window. 
![polyglot logohraph enhancement](https://user-images.githubusercontent.com/46006628/50252031-56ad4e00-03ab-11e9-9aeb-40066c29697a.png)
well firstly related words are kinda slow to add so you could add quick buttons as show below (with the other feature)
and a in-app editor for logographs (only a preview/Base Idea image):
![polyglot logohraph enhancement edit](https://user-images.githubusercontent.com/46006628/50252028-557c2100-03ab-11e9-91a1-e1312efe6330.png)
With some features such as...
![logograph editor 0](https://user-images.githubusercontent.com/46006628/50255165-65e5c900-03b6-11e9-8d13-662cf5a03c36.png)
![logograph editor 2](https://user-images.githubusercontent.com/46006628/50255609-320ba300-03b8-11e9-91dd-0dfc19e614a7.png)
Selecting tools, draw tools and such and a stroke order feature that works counting the colored strokes you make to put it in order. (it is also able to change the order manually)
![logograph editor 4](https://user-images.githubusercontent.com/46006628/50255344-194ebd80-03b7-11e9-824f-e5ebecaf2c24.png)
![logograph editor 1](https://user-images.githubusercontent.com/46006628/50255607-320ba300-03b8-11e9-8b9c-4c7cfc4a1ac7.png)
and a radical system to make new characters such as japanese kanji, korean hangeoul ETC.
![logograph editor 3](https://user-images.githubusercontent.com/46006628/50255606-31730c80-03b8-11e9-9b7e-28bfb609c41e.png)

and if this is too much you can only add the related word thing please? thank you
PD: I do love your work on this project keep it up!

**Comment by DraqueT** [comment]
Thank you for your interest about PolyGlot and your ideas! It's great when users let me know how things are working for them and what features would be useful! :3

Could you explain the related word feature that you're asking for a bit more? That is one that I might be able to implement for you. Your idea for an in-program editor is a good one and I'll have to give it some thought. It would have to be something that is free and open source though, as making something like that from scratch would be a bit more than I have time for right now.

**Comment by Fcomega121** [comment]
That's right! so my idea for relating logographs is pretty simple, it's just about 2 things:
1. make directly word relations from main logo's window..
![related wordlogograph 2](https://user-images.githubusercontent.com/46006628/50379283-89558180-060b-11e9-8ad8-27fd7fff2562.png)
2. the actual system is a bit confusing you have to open the RW-L window and click on add new logograph and then click again that button...
![related wordlogograph 1 edit](https://user-images.githubusercontent.com/46006628/50379292-e3eedd80-060b-11e9-88b2-2c6708406a5c.png)
so it should be most intuitive to avoid confusion and make it most easy like this:
![related wordlogograph 1 edit01](https://user-images.githubusercontent.com/46006628/50379300-11d42200-060c-11e9-80a3-4dfbc079bb6d.png)
I hope i've explained it well cause my english is a bit rusty since 1 year ago.

**Comment by DraqueT** [comment]
You've explained it well, thank you! I need to put some thought into this. These are all great ideas, and I might be able to put some of them into PolyGlot in 2.5 (unfortunately they won't make it into 2.4).

The visual guides you've included show me very clearly what you're thinking of, thanks for taking the time to put them together. :3

**Comment by Fcomega121** [comment]
thank you for the comments! It was a pleasure to make them, I do love to make visual guides as well. ^v^
